docs/proposed/grid-manager/managed-grid.rst

```diff
@@ -0,0 +1,338 @@
+   to re-start. In the Future(tm), we should consider (in exarkun's
+   words):
+
+       "A python program you run as a new process" might not be the
+       best abstraction to layer on top of the configuration
+       persistence system, though.  It's a nice abstraction for users
+       (although most users would probably rather have a GUI) but it's
+       not a great abstraction for automation.  So at some point it
+       may be better if there is CLI -> public API -> configuration
+       persistence system.  And maybe "public API" is even a network
+       API for the storage server so it's equally easy to access from
+       an agent implemented in essentially any language and maybe if
+       the API is exposed by the storage node itself then this also
+       gives you live-configuration-updates, avoiding the need for
+       node restarts (not that this is the only way to accomplish
+       this, but I think it's a good way because it avoids the need
+       for messes like inotify and it supports the notion that the
+       storage node process is in charge of its own configuration
+       persistence system, not just one consumer among many ... which
+       has some nice things going for it ... though how this interacts
+       exactly with further node management automation might bear
+       closer scrutiny).
+
+
+Enrolling a Storage Server: Config
+----------------------------------
+
+You may edit the ``[storage]`` section of the ``tahoe.cfg`` file to
+turn on grid-management with ``grid_management = true``. You then must
+also provide a ``[grid_management_keys]`` section in the config-file which
+lists ``name = path/to/certificate`` pairs.
+
+These certificate files are issued by the ``tahoe grid-manager sign``
+command; these should be **securely transmitted** to the storage
+server. Relative paths are based from the node directory. Example::
+
+    [storage]
+    grid_management = true
+
+    [grid_management_keys]
+    default = example_grid.cert
+
+This will cause us to give this certificate to any Introducers we
+connect to (and subsequently, the Introducer will give the certificate
+out to clients).
+
+
+Enrolling a Client: CLI
+-----------------------
+
+DECIDE: is a command like this best, or should you have to edit the
+        config "by hand"? (below fits into warner's philosophy that "at some
+        point" it might be best to have all config in a database or similar
+        and the only way to view/edit it is via tahoe commands...)
+
+tahoe add-grid-manager
+``````````````````````
+
+- ``--name``: a petname to call this Grid Manager (default: "default")
+
+For clients to start using a Grid Manager, they must add a
+public-key. A client may have any number of grid-managers, so each one
+has a name. If you don't supply ``--name`` then ``"default"`` is used.
+
+This command takes a single argument, which is the hex-encoded public
+key of the Grid Manager. The client will have to be re-started once
```

Comment:
I clarified this (because yeah it's just the output of `public-identity` not some extra encoding, and that encoding isn't just "hex", it's ... whatever it is ;) )